assets/js/press-sync.js

```diff
@@ -2,6 +2,8 @@ window.PressSync = ( function( window, document, $ ) {
 
 	var app = {};
 
+	app.PAGE_SIZE = 1;
```

Comment:
What does PAGE_SIZE represent? It looks like it's used for calculating time below, but we're dividing by 1, which is just an extraneous calculation. Do we actually need this?